### PR TITLE
Increase color palette to 20 colors.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

**Comment by inferno-chromium** [comment]
colors can be seen here - https://public.tableau.com/views/TableauColors/ColorPaletteswithRGBValues?%3Aembed=y&%3AshowVizHome=no&%3Adisplay_count=y&%3Adisplay_static_image=y
mix of dark and light colors.

File: analysis/plotting.py [hunk 1]
```diff
@@ -61,14 +61,21 @@ def _formatted_title(benchmark_snapshot_df):
...
 
     def __init__(self, fuzzers, quick):
         """Instantiates plotter with list of |fuzzers|. If |quick| is True,
         creates plots faster but, with less detail.
         """
-        palette = sns.color_palette(n_colors=len(fuzzers))
         self._fuzzer_colors = {
-            fuzzer: palette[idx] for (idx, fuzzer) in enumerate(sorted(fuzzers))
+            fuzzer: self.COLOR_PALETTE[idx % len(self.COLOR_PALETTE)]
+            for (idx, fuzzer) in enumerate(sorted(fuzzers))
```
Comment (jonathanmetzman) [inline analysis/plotting.py]: Can we eliminate the paren around `idx, fuzzer`? Seems nonstandard.

File: analysis/plotting.py [hunk 2]
```diff
@@ -61,14 +61,21 @@ def _formatted_title(benchmark_snapshot_df):
 
 class Plotter:
     """Plotter that uses the same color for the same fuzzer."""
+    # Tableau 20 colors.
+    COLOR_PALETTE = [
```
Comment (lszekeres) [inline analysis/plotting.py]: Consider using the "private" naming convention: _COLOR_PALETTE

File: analysis/plotting.py (hunk 1 above)
Comment (inferno-chromium) [reply analysis/plotting.py]: Done!

File: analysis/plotting.py (hunk 2 above)
Comment (inferno-chromium) [reply analysis/plotting.py]: Done!

**Pull request merged**